app/models/user.rb

```diff
@@ -5,4 +5,21 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
 
   has_one_attached :avatar
+
+  has_many :active_friendships, class_name: 'Friendship', foreign_key: 'follow_source_id', dependent: :destroy, inverse_of: :follow_source
+  has_many :passive_friendships, class_name: 'Friendship', foreign_key: 'follow_target_id', dependent: :destroy, inverse_of: :follow_target
+  has_many :followings, through: :active_friendships, source: :follow_target
+  has_many :followers, through: :passive_friendships, source: :follow_source
+
+  def follow(other_user)
+    active_friendships.create(follow_target_id: other_user.id)
```

Comment:
ここもオブジェクトをそのまま渡せるかもしれません。

```suggestion
    active_friendships.create(follow_target: other_user)
```

Reply:
引数に直接idを渡すように修正します！